test/setup

```diff
@@ -57,7 +57,7 @@ echo "booting ${VMS} by heat, then copy files for pre-deployment"
 #sleep 30
 extra_args="--ursula-debug --provisioner=heat --heat-stack-name=${testenv_heat_stack_name} --ursula-user=${LOGIN_USER} --heat-parameters ${EXTRA_HEAT_PARAMS}"
 ursula ${extra_args} "envs/test" ${ROOT}/playbooks/${TEST_ENV}/tasks/pre-deploy.yml -u ${LOGIN_USER} "$@" -vvvv
-
+cp envs/test/.ssh_config  envs/test/ssh_config
```

Comment:
can't we have just one file? i.e envs/test/ssh_config? Who relies on .ssh_config and can we fix that?